fix(GetConstraintsPertab): Also work with non-lowercase column names

janpio opened the pull request:
Possibly closes #6

(Sorry about the linebreak thing at the end of the file, GH UI doesn't let me not have one when I edit files there.)

janpio committed:
fix(GetConstraintsPertab): Also work with non-lowercase column names

Possibly closes #6

faisaltheparttimecoder commented:
@janpio will it be possible to merge this fix on your branch (https://github.com/janpio/mock-data/pull/1), once done I will have this pull request squashed onto master branch for a new release

faisaltheparttimecoder committed:
Fix for uppercase column (#1)

* Fix for uppercase colum

+ Uppercase columns are being converted to lowercase and hence error'ng out. this is fixed
+ Bump the version of the mock-data tool

* version bump

janpio commented:
Done.

Still a bit confused how the problem I tried to solve is solved via the change you added, but - if it works it works 🤷‍♀

faisaltheparttimecoder commented:
The issue is the columns case sensitivity is kept till it reached the copy statement, during the execution postgres replaces the uppercase with lower case, so added double quotes at copy statement for column to keep the case sensitivity.
